spec/factories/staff.rb

```diff
@@ -53,8 +54,12 @@
       confirmed_at { Time.zone.now }
     end
 
-    trait :assessor do
-      assessor { true }
+    trait :award_decline do
```

Comment:
Shall we call these `with_award_decline` and `with_support_console_access`? Reads a bit better. Possibly even with a `_permission` suffix?

Reply:
I think the suggestion here was just to change the factory trait name rather than the database field, but actually I like the `_permission` prefix on the database field names but I don't think we need the `with_`, so something like:

```rb
# using the trait
create(:staff, :with_award_decline_permission)

# which would be equivalent to
create(:staff, award_decline_permission: true)
```